src/libs/ajax/SamResources.ts

```diff
@@ -51,6 +51,13 @@ export const SamResources = (signal?: AbortSignal) => ({
       _.mergeAll([authOpts(), { signal }])
     ).then((r) => r.json());
   },
+
+  canDelete: async (fqResourceId: FullyQualifiedResourceId): Promise<boolean> => {
```

Comment:
We have no way to tell if the user was an Owner on the workspace without checking directly with Sam permissions-- details in Slack [thread](https://broadinstitute.slack.com/archives/C02L9L96YB1/p1726606244226999).

Reply:
The slack thread is in a private channel so tl;dr for posterity: 

Rawls replaces the user's access level on the workspace with `No Access` if they are not a member of any auth domain groups ([source](https://github.com/broadinstitute/rawls/blob/712a4ac2f869ee068a4459998af52f72a94a2df2/core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala#L961-L966)), but the `delete` action in Sam isn't `authDomainConstrainable` ([source](https://github.com/broadinstitute/sam/blob/4564928c920ad1776a73a80bf4af22e345f9d422/src/main/resources/reference.conf#L60-L62)) which means that Sam will allow a user to delete a workspace even if they aren't in all of the workspace's auth domain groups. Since Rawls drops the user's actual access level in this case, it's best for the UI to check with sam directly to see if the user should be able to delete the workspace. Rawls will still double check with Sam before allowing the user to delete the workspace.